tests/Tests/200__monitor_and_manage/200__metrics/204_check_pager_duity.robot

```diff
@@ -0,0 +1,49 @@
+*** Settings ***
+Documentation   RHODS_Pager_Duity_key__VERIFICATION
+...             Verify that rhods operator log is clean and doesn't contain any error regarding resource kind
+...
+...             = Variables =
+...             | NAMESPACE                | Required |        RHODS Namespace/Project for RHODS operator POD |
+...             | REGEX_PATTERN            | Required |        Regular Expression Pattern to match the erro msg in capture log|
+...             | Secret Name              | Required |        Secret Name|
+...             | CONFIGMAP_NAME           | Required |        Name  of config map|
+
+Library        Collections
+Library        OperatingSystem
+Library        String
+Library        OpenShiftCLI
+
+
+*** Variables ***
+${NAMESPACE}            redhat-ods-monitoring
+${CONFIGMAP_NAME}         alertmanager
+${REGEX_PATTERN}          service_key(:).*
+${SECRET_NAME}          redhat-rhods-pagerduty
+
+
+*** Test Cases ***
+PagerDuty Dummy Secret Verification
+     [Documentation]    Verification of PagerDuty Secret
+     [Tags]  Sanity
+     ...     ODS-737
+     ${service_key}   Get Service Key From Alertmanager ConfigMap
+     ${secret_key}    Get Secert Key From Secrets
+     Should Be Equal As Strings    ${service_key}   ${secret_key}
```

Comment:
In Polarion  title is ODS-737 - Verify PagerDuty dummy secret is correctly stored in when installing RHODS **not** using the add-on flow

That means that:
- You need to add the tag **Deployment-Cli**  (I just added for you in the style guide)
- The secret will be always the same, so you can check the value of it to verify it's a dummy one. Maybe you can add this step to Polarion 